docs/cloud.md

```diff
@@ -0,0 +1,55 @@
+# Deploying Antrea on a cloud provider
+
+Antrea may run in networkPolicyOnly mode in some cloud managed clusters. This document describes
+ steps to create EKS using terraform.
```

Comment:
```suggestion
steps to create EKS using terraform.
```

Reply:
do think I will need to make this change, the rendered output is

" ... This document describes steps to create EKS using terraform."

Reply:
It's funny that your text editor formats it this way, I don't recall ever seeing a whitespace at the beginning of a line in a markdown document, but that's very minor